docs/user-guide/02-convenience-features.md

```diff
@@ -0,0 +1,227 @@
+# Basic Convenience Features
```

Comment:
Minor nit: stylistically, I find that the "ATX-style" headers are nicer to look at when you have the cases they cover (1st- and 2nd-level headers):

```
First Level
===========

Second Level
------------
```

They will render the same in the output HTML, but when reading or editing the plain text of the Markdown file, they tend to stand out more and make it easier to visually parse the document structure.